presets/4.3/rc_link/IRC_race.txt

```diff
@@ -6,7 +6,7 @@
 #$ AUTHOR: sugarK
 #$ DESCRIPTION: Setup for the various ImmersionRC modes via GHST protocol for racing.
 #$ DESCRIPTION: WARNING: Make ABSOLUTELY SURE that the Hardware > ADC Filter is un-checked! and both radio and the TX unit are all up-to-date with the latest EdgeTX, OpenTX or FreedomTX.
-#$ DESCRIPTION: For racing it makes sense to use one of the 250/500Hz modes which the 'race' mode is only 166hz.
+#$ DESCRIPTION: For racing it makes sense to use one of the 250/500Hz modes which the 'race' mode is only 166Hz.
```

Comment:
I don't understand this line.  The first part is true: "For racing it makes sense to use one of the 250/500Hz modes" is correct.  The second part , "which the 'race' mode is only 166Hz." seems to me too make no sense and should be deleted, or updated to say whatever was intended.

Reply:
should be while 'Race' mode is only 166Hz

Reply:
maybe `, because the 'race' mode is only 166Hz.`

Reply:
That works

Reply:
`#$ DESCRIPTION: For racing it makes sense to use one of the 250/500Hz modes, because the 'race' mode is only 166Hz.`

Reply:
modified it like that:
#$ DESCRIPTION: 250Hz and 500Hz modes have less delay than the 'race' mode, which is only 166Hz.
Because 250Jz and 500Hz gives less latency regardless if thats for racing or not :)